test/end-to-end/cases/expected/xquery-tutorial-result-norm.html

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?><html xmlns:test="http://www.jenitennison.com/xslt/unit-test" xmlns="http://www.w3.org/1999/xhtml">
    <head>
       <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-      <title>Test Report for www.functx.com (passed: 1 / pending: 0 / failed: 0 / total: 1)</title>
+      <title>Test Report for http://www.functx.com (passed: 1 / pending: 0 / failed: 0 / total:
+         1)
```

Comment:
This is a minor issue: it seems to me that this extra formatting with whitespace is unnecessary and makes the title less readable - maybe it was due to automatic formatting? I think keeping the original version would be better.

Reply:
We need to keep the extra whitespace, because that's what Saxon 9.8 and 9.7 actually serialize the `title` element. You can see it with the following dumb stylesheet.

**test.xsl**
```xml
<?xml version="1.0" encoding="UTF-8"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="3.0">
   <xsl:template name="main">
      <xsl:message select="('xsl:product-name', 'xsl:product-version') ! system-property(.)" />
      <html xmlns:test="http://www.jenitennison.com/xslt/unit-test" xmlns="http://www.w3.org/1999/xhtml">
         <head>
            <title>Test Report for http://www.functx.com (passed: 1 / pending: 0 / failed: 0 / total: 1)</title>
         </head>
      </html>
   </xsl:template>
</xsl:stylesheet>
```

```
C:\>java -jar saxon9ee.jar -it:main -xsl:test.xsl
SAXON EE 9.7.0.21
<?xml version="1.0" encoding="UTF-8"?><html xmlns:test="http://www.jenitennison.com/xslt/unit-test" xmlns="http://www.w3.org/1999/xhtml">
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
      <title>Test Report for http://www.functx.com (passed: 1 / pending: 0 / failed: 0 / total:
         1)
      </title>
   </head>
</html>
```

It seems fixed on 9.9. ([Bug 3842](https://saxonica.plan.io/issues/3842) perhaps?)
```
P:\>java -jar saxon9ee.jar -it:main -xsl:test.xsl
SAXON EE 9.9.0.2
<?xml version="1.0" encoding="UTF-8"?><html xmlns:test="http://www.jenitennison.com/xslt/unit-test" xmlns="http://www.w3.org/1999/xhtml">
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
      <title>Test Report for http://www.functx.com (passed: 1 / pending: 0 / failed: 0 / total: 1)</title>
   </head>
</html>
```

One day when 9.9 is considered as the mainstream, we can probably update the HTML files in the repository by regenerating them using 9.9. But it's not now, so the extra whitespace is there.

Reply:
I see. That's absolutely fine, thank for the explanation. I'm merging this pull request into `master`.